app/assets/stylesheets/components/_btn.scss

```diff
@@ -51,7 +51,7 @@
   &:active,
   &:focus,
   &:hover, {
-    border: 0;
+    //border: 0;
```

Comment:
By changing the link into a button, the button loses its border when you hover over it, and I wasn't sure how to fix that aside from commenting this out. Let me know if there's a better way to do that.

Reply:
Why didn't we need this for the other uses of `button_to` in the app? Unsure if we need it